### ISSUE TITLE
Adopt some of the clean-up from dhs-ncats/gatherer

### ISSUE THREAD

**Issue by jsf9k** [18F/domain-scan#288]
In dhs-ncats/gatherer we do some cleaning up of the domains returned by this project's `gather` pipeline.  (For example, we remove `\r` characters that appear in the middle of host names and we condense repeated `.` characters into a single one.)  Does it make sense to do some of those things here instead?

**Comment by konklone** [comment]
Yes, I think it would totally make sense to do that here, since these all represent generic errors in hostname representation.
